atomicapp/nulecule/main.py

```diff
@@ -228,6 +228,10 @@ def run(self, cli_provider, answers_output, ask,
         if not self.answers_file:
             self._process_answers()
 
+        # Give more prefernce to the provider that is given via CLI
```

Comment:
@cdrage '*when*' vs '*that*' completely changes the meaning of the sentence.

if '*when*' is used:
Meaning becomes like "Give more preference to provider" (BTW we always give preference to provider, be it any; provided from anywhere)

if '*that*' is used:
Meaning becomes "Give more preference to provider which is mentioned on the CLI", this also means that user might have specified "provider" in answers.conf or something